package.json

```diff
@@ -157,8 +157,7 @@
     "temp": "^0.9.0",
     "text-buffer": "13.17.0",
     "timecop": "https://www.atom.io/api/packages/timecop/versions/0.36.2/tarball",
-    "tree-sitter": "0.15.6",
-    "tree-sitter-css": "^0.13.7",
```

Comment:
The `tree-sitter-css` module shouldn't be listed as a direct dependency here. I added that by mistake; it's only used by `language-css` module, which [depends](https://github.com/atom/language-css/blob/09c09b5b9d933b6b5a1591a3c954ec1829b2149e/package.json#L25) on it explicitly.